### ISSUE TITLE
Credentials are exposed when closing a IdaDataBase connections

### ISSUE THREAD

**Issue by shafeneger** [ibmdbanalytics/ibmdbpy#14]
I am using `ibmdbypy`on IBM Data Science Experience. 
When I close a `IdaDataBase` connection is closed I can see the entire ODBC connection URL including the user name and password.
Is there a way to disable it? DSX has a filtering options to exclude credentials during sharing a notebook via a public URL. However, if the ODBC URL is exposed during closing of a connection it makes it a little bit more impractical for sharing (when credentials are sensitive information).

**Comment by shafeneger** [comment]
The exact output message is the following (actual credentials removed!):
```
A SQL-Handle for database DASHDB;Database=BLUDB;Hostname=<HOSTNAME>;Port=50000;PROTOCOL=TCPIP;UID=<USERNAME>;PWD=<PASSWORD> was closed.
```

**Comment by nicoleschoen** [comment]
Thanks Sven, will look into that one first.

**Comment by nicoleschoen** [comment]
Will now only show "Connection closed", find fix in ibmdbpy 0.1.3.post8
